sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentAnalysisClient.cs

```diff
@@ -97,8 +97,10 @@ public DocumentAnalysisClient(Uri endpoint, TokenCredential credential, Document
 
             options ??= new DocumentAnalysisClientOptions();
 
+            string defaultScope = $"{(string.IsNullOrEmpty(options.Audience.ToString()) ? DocumentAnalysisAudience.AzurePublicCloud : options.Audience)}/.default";
```

Comment:
```suggestion
            string defaultScope = $"{(string.IsNullOrEmpty(options.Audience?.ToString()) ? DocumentAnalysisAudience.AzurePublicCloud : options.Audience)}/.default";
```

We need to account for the audience being `null`.